web/sections/FactCore/Fact.tsx

```diff
@@ -0,0 +1,149 @@
+import { toPlainText } from '@portabletext/react'
+import type { PortableTextBlock } from '@portabletext/types'
+import type { BackgroundColours, ImageWithAlt } from '../../types/index'
+import { urlFor } from '../../common/helpers'
+import Blocks from '../../pageComponents/shared/portableText/Blocks'
+import { Typography } from '@core/Typography'
+import { forwardRef, HTMLAttributes } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { BackgroundContainer, BackgroundContainerProps } from '@components/Backgrounds'
+import { ColorKeyTokens } from '../../styles/colorKeyToUtilityMap'
+import Image from '../../pageComponents/shared/SanityImage'
+
+type FactImagePosition = 'left' | 'right';
+
+type FactboxProps = {
+  title: string
+  content: []
+  background: { title: string; value: string; key?: string; dark?: boolean }
+  image: ImageWithAlt
+  imagePosition: FactImagePosition
+  dynamicHeight: boolean
+  isSingleColumn?: boolean
+}
+
+type BlockProps = {
+  isInline: boolean
+  value: FactboxProps
+  className?: string
+} & PortableTextBlock
+
+export type FactProps = {
+  background?: string
+  backgroundUtility?: string
+  dark?: boolean
+  imagePosition?: FactImagePosition
+  useTwoColumns?: boolean
+} & HTMLAttributes<HTMLDivElement>
+
+export const Fact = (block: BlockProps) => {
+  const { value, className } = block
+  const { title, content, background, image, imagePosition, isSingleColumn } = value
+
+  const bgTitle = (background ? background?.title : 'White') as BackgroundColours
+  if (!content || content.length === 0) {
+    return null
+  }
+
+  const imageSrc = image && image.asset ? urlFor(image).size(1200, 800).auto('format').toString() : false
+
+  const plainText = content ? toPlainText(content as PortableTextBlock[]) : ''
+
+  const hasColumns = !imageSrc && !isSingleColumn && plainText.length > 800
+  const hasImage = imageSrc ? true : false
+  const hasBgColor = bgTitle !== 'White'
+
+  const ImageComponent = ({ imagePosition = 'left' }: { imagePosition?: FactImagePosition }) => (
```

Comment:
Image position is param is not used. Could you check if this works when image position is set to right?